package.json

```diff
@@ -17,7 +17,7 @@
     "test:watch": "karma start --auto-watch --browsers=ChromeHeadless",
     "test:pkgs": "tools/scripts/test.pkgs.sh",
     "test:pkgs:minified": "tools/scripts/test.pkgs.minified.sh",
-    "test:types": "tsc",
+    "test:types": "tsc --typeRoots [\"./types\"]",
```

Comment:
Shouldn't that be part of our tsconfig?
Might be best not to spread configuration around if possible.

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html

Reply:
done